### PR TITLE
Slightly Better TimePicker

### PR DESCRIPTION
**Pull request by EliteAsian123**
## Types of Changes 
  
 - [ ] Bug fix (non-breaking change that fixes an issue) 
 - [x] New feature (non-breaking change that adds functionality) 
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
 

### Changes

- [x] Bug fix: Added padding to the seconds so it displays `00` instead of just `0`
- [x] Adjustment: The time updates *while* the slider is being moved instead of after it has moved
- [x] Adjustment: Slider ticks 

## Screenshots (if appropriate):
**Before:**<br>
Before

**After:**<br>
After

### CODE REVIEW

**Review by BilelJegham — CHANGES_REQUESTED** [review]
Hello, thanks for you contribution.

I write a small feedback.

Best regards,
Bilel

File: src/components/shared/TimePicker.vue [hunk 1]
```diff
@@ -58,6 +59,17 @@ export default {
             // Tape in input second
             this.$emit('input', Math.floor(this.value / 60) * 60 + parseInt(s)); // Get number minutes and add seconds
         },
+        getSeconds() {
```
[the remark this answers is not in the document]
Comment (EliteAsian123) [reply src/components/shared/TimePicker.vue]: Yes, it should be. Sorry, I'm newer to Vue.

**Review by BilelJegham — APPROVED** [review]
Looks good for me.

Thanks @EliteAsian123 


@all-contributors please add for code

**Comment by BilelJegham** [comment]
@all-contributors please add @EliteAsian123 for code

**Pull request merged**